lib/syncthing/syncthing.go

```diff
@@ -475,3 +482,37 @@ func (e *controller) Shutdown() {
 func (e *controller) ExitUpgrading() {
 	e.Stop(ExitUpgrade)
 }
+
+type supervisor interface{ Services() []suture.Service }
```

Comment:
Why do you need that type instead of using `*suture.Supervisor` directly?

Reply:
We typically wrap the supervisor so the concrete type we’re looking at at any given moment isn’t a *suture.Supervisor. The top level is but not when we’ve dug into the tree.